nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestWaitNotifyProtocol.java

```diff
@@ -104,8 +105,10 @@ public void testNotifyFirst() throws Exception {
 
         final AtomicCacheEntry<String, String, Long> cacheEntry = cacheEntries.get("signal-id");
 
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
The `ObjectMapper` instance can be created as a static variable and reused in all methods.